test/csuite/random_abort/main.c

```diff
@@ -612,21 +612,22 @@ main(int argc, char *argv[])
     WT_RAND_STATE rnd;
     pid_t pid;
     uint32_t i, j, nth, timeout;
-    int ch, status;
+    int ch, status, ret;
     char buf[1024], fname[MAX_RECORD_FILES][64];
     const char *working_dir;
-    bool rand_th, rand_time, verify_only;
+    bool preserve, rand_th, rand_time, verify_only;
 
     (void)testutil_set_progname(argv);
 
     compaction = compat = inmem = false;
     nth = MIN_TH;
+    preserve = false;
     rand_th = rand_time = true;
     timeout = MIN_TIME;
     verify_only = false;
     working_dir = "WT_TEST.random-abort";
 
-    while ((ch = __wt_getopt(progname, argc, argv, "Cch:mT:t:v")) != EOF)
+    while ((ch = __wt_getopt(progname, argc, argv, "Cch:mpT:t:v")) != EOF)
```

Comment:
not a problem so much as a general question - most of these flags seem to be missing from the `usage()` function above. is that because we don't really expect this to be used interactively?

Reply:
I expect this is just an oversight. I'll investigate and correct accordingly.

Reply:
I think you're right about these not being used interactively. I can only see them being called as part of `make check`. I think I'll leave `usage()` alone for now instead of digging through the code to determine how each of the flags is used.

Reply:
yep, sounds good! thanks for checking